src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Styles/_colors.scss

```diff
@@ -1,21 +1,4 @@
-﻿$PrimaryColor: #1C5BFF;
-$SecondaryColor: #233184;
-$OrangeColor: #EF9C00;
-//============
-$DarkBlue: #2F455A;
-$Green: #14A38B;
-$Yellow: #EF9C00;
-$Orange: #E39F0B;
-$Blue: #2156A6;
-$DarkGreen: #064006;
-$DarkOrange: #D4783F;
-$DarkYellow: #E0CC12;
-$LightBlue: #63A5DB;
-$LightGreen: #2E9952;
-$LightPurple: #CB42F5;
-//=============================================================
-
-$B1Color: #0054C6;
+﻿$B1Color: #0054C6;
```

Comment:
What's going on here? are these colors redundant?

Reply:
These colors are redundant and I had removed them before.
But I think the changes have been missed in a merge conflict.